docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,8 +1,55 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-# Local agents
\ No newline at end of file
+# Running a Dagster+ agent locally
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) locally. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent (like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
```

Comment:
I think we should put some of this wording (or add a warning, including adding the drawbacks more explicitly) in a `:::warning` box, like:
```markdown
:::warning
Local agents are not well suited for production use cases. This is because local agents:

* do not have the same CI / CD update strategies as the other agents
* need a separate process running on the sever hosting the agent to pull changes onto it
* need to restart the agent to deploy changes 

:::
```

The non-development use case I have seen for local agents is to use them when you need to co-locate your agent alonside some sensitive data that cannot leave that server, not sure if it's worth mentioning but it might help to say "this is a very special tool to use in specific circumstances but should not be considered as a general way to execute dagster"